predicting_wine_cultivars.ipynb

```diff
@@ -307,11 +307,12 @@
    "source": [
     "## Methods & Results\n",
     "---\n",
-    "1. describe in written english the methods you used to perform your analysis from beginning to end that narrates the code the does the analysis.\n",
-    "2. your report should include code which:\n",
-    "3. loads data from the original source on the web\n",
-    "4. wrangles and cleans the data from it’s original (downloaded) format to the format necessary for the planned classification or clustering analysis\n",
-    "5. performs a summary of the data set that is relevant for exploratory data analysis related to the planned classification analysis\n",
+    "This project utilized a linear regression model to understand the effects of various chemical properties in order to predict what cultivar a wine was derived from. First, we read in data from the [UCI Machine Learning Repository](https://doi.org/10.24432/C5PC7J). It contains data about various wines from Italy derived from three different cultivars. Each row represents the chemical and physical properties of a different wine, such as its concentration of alcohol, magnesium level and hue.\n",
```

Comment:
I think we should use k-means clustering or something similar instead of linear regression because the cultivars are categorical variables